setup.py

```diff
@@ -39,13 +39,11 @@ def read(fname):
     ],
     install_requires=[
         'Jinja2>=2.8',
-        'pytz==2017.2',
         'PyYAML>=3.11',
-        'psycopg2==2.7.5',
+        'psycopg2>=2.7.5,<2.8',
         'sqlparse==0.2.3',
         'networkx==1.11',
-        'snowplow-tracker==0.7.2',
-        'celery==3.1.23',
+        'minimal-snowplow-tracker==0.0.1',
         'snowflake-connector-python>=1.4.9',
         'requests>=2.18.0',
```

Comment:
This should probably be `>=2.18.0,<3` - requests is very good about semver but that means any `3.0` will be pretty much guaranteed to break something...